Gopkg.toml

```diff
@@ -46,7 +46,7 @@ version="v1.4.7"
 
 [[constraint]]
   name = "knative.dev/serving"
-  version = "=v0.8.0"
```

Comment:
Let's make this change after we release 0.2.1 ?  A bunch of fixes need to get into that and manifests are still on knative 0.8.

Reply:
@yuzisun what are we waiting for in 0.2.1?